src/app/screens/Receive/index.tsx

```diff
@@ -223,7 +238,7 @@ function Receive() {
         title={t("title")}
         headerLeft={
           <IconButton
-            onClick={() => navigate("/")}
+            onClick={handleBack}
```

Comment:
Also, we can use it like this here:
```
onClick={() => {
    invoice ? setDefaults() : navigate("/");
}}

and
```
onClick={() => {
    setDefaults();
    navigate("/receive");
}}

to avoid having `handleBack` and `receiveAnotherPayment` functions

Reply:
Done! @im-adithya